package edu.nf.ViPoPhone.exception;

public class UsersException extends RuntimeException{

	public UsersException(String message, Throwable cause) {
		super(message, cause);
		// TODO 自动生成的构造函数存根
	}

	public UsersException(String message) {
		super(message);
		// TODO 自动生成的构造函数存根
	}

	public UsersException(Throwable cause) {
		super(cause);
		// TODO 自动生成的构造函数存根
	}

	

}

